# HG changeset patch
# Parent 52eb582f82c5d8ac474731324eb3cd14e69c0e65
Cleanup double appearances of (status) in help output

diff -r 52eb582f82c5 etc/rc.d/ipfilter
--- a/etc/rc.d/ipfilter
+++ b/etc/rc.d/ipfilter
@@ -23,7 +23,7 @@
 resync_cmd="ipfilter_resync"
 status_precmd="$stop_precmd"
 status_cmd="ipfilter_status"
-extra_commands="reload resync status"
+extra_commands="reload resync"
 required_modules="ipl:ipfilter"
 
 ipfilter_start()
diff -r 52eb582f82c5 etc/rc.d/pf
--- a/etc/rc.d/pf
+++ b/etc/rc.d/pf
@@ -19,7 +19,7 @@
 reload_cmd="pf_reload"
 resync_cmd="pf_resync"
 status_cmd="pf_status"
-extra_commands="check reload resync status"
+extra_commands="check reload resync"
 required_files="$pf_rules"
 required_modules="pf"
 
